src/userApp/docs.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @swagger
+ * /api/v1/users/reset/email:
```

Comment:
change the documentation URLs to point to the right routes since we shifted to the use of 
`/accounts/forgot-password` on this route